### DISCUSSION TITLE
An option to hide beatmaps you already have in the beatmap search

### DISCUSSION THREAD

**Discussion by Doksha** [ppy/osu#15688] [Ideas] [answered]
This feature is already in stable and would be a nice addition to lazer's version of direct, trying to manually look for beatmaps I do not have is very time consuming, no matter how I go about it (stable, 3rd party websites, lazer, etc.) and this could vastly improve user experience

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/15727

**Comment by peppy** [comment]
@Doksha see https://github.com/ppy/osu/issues/15727#issuecomment-974962193 for reasoning as to why the UX is hard to implement, and not super great on stable too (only works up to a certain point).

Have you tried using the "played" filter as an alternative? It's supporter only but provides similar functionality.
